### ISSUE TITLE
About data preprocessing!!

### ISSUE THREAD

**Issue by Z-Z-J** [redrock303/HEMlets#2]
Thank you for your great work! I want to know the method of data preprocessing。Can you provide the relevant code？

**Comment by redrock303** [comment]
Sorry for the very later reply, we have not released the training code yet. But if you have any questions about our work, you can commit your specific problems. We will try our best to solve them. In general, we follow the [Intergal Human Pose](https://github.com/JimmySuen/integral-human-pose) for data processing. Here, I give some fragments for a quick overview of our data pre-processing:
(1) using gt calibration information for obtaining the ground truth bounding box only, where the box's center is the root joint.
rect2d_l, rect2d_r, rect2d_t, rect2d_b, pt_2d, pt_3d, vis, pelvis3d = from_worldjt_to_imagejt(Joint3d =joint3d_j18,rot = R,trans = T,fl =f,c_p = c,rect_3d_width=2000,
        rect_3d_height= 2000 , joint_num=18 )
(2) we normalize the depth to (-1,1) with a maximum height assumption of 2000mm：
cropp2dPose3d[:,2] = (pt_2d[:,2] / 2000.0 + 0.5) * 255.0.

**Comment by wyx1997** [comment]
@redrock303 Hi, Thank you for your great work! I have two questions for data processing. 
The first is that you mean you normalize the depth to (-1,1), do you do centralization before it? And why you add 0.5 to the quotient?
The second is that how you do data processing for MPII data. I mean if the x and y coordinates should also divide something for Loss 3d?
Looking forward for your reply! Thanks a lot!
